### PR TITLE
Marathon Docs for Checks

### PR DESCRIPTION
**Pull request by kensipe**
Summary:
Documentation for Checks

JIRA issues:

### CODE REVIEW

File: docs/docs/checks.md [hunk 1]
```diff
@@ -0,0 +1,101 @@
+---
+title: Checks
+---
+
+# Checks
+
+Marathon already has the concept of [health checks]({{ site.baseurl }}/docs/health-checks.html) and [readiness checks]({{ site.baseurl }}/docs/readiness-checks.html), which either monitor the health of an application or provide an indication that the service is ready.  Both of these features in Marathon will indicate that the app/pod is "in deployment".
+
+Checks provide an abstract way to check on a task in Mesos providing values back via tasks and podstatus which are *not* interpreted by Marathon.   They are intended to be consumed by services external to Marathon which will take appropriate action on them.   You can think of them as a health check without a kill policy.  The most significant aspect of them is that Marathon does not evaluate anything about the return values of a check.
```
Comment (alenkacz) [inline docs/docs/checks.md]: I would link mesos docs here

File: docs/docs/checks.md [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+
+Checks can be defined as:
+
+* HTTP Check
+* TCP Check
+* Command Check
+
+When a check is configured for an app/pod, it has zero impact on its deployment status in Marathon.  To control this behavior you will need to use a health-check or readiness check.  The value in the task varies depending on check type:
```
Comment (alenkacz) [inline docs/docs/checks.md]: health check not health-check

File: docs/docs/checks.md (hunk 1 above)
Comment (timcharper) [reply docs/docs/checks.md]: Using terminology "already has the concept" sounds like terminology for a changelog. I think this page should be written from the perspective of what is, not what was.

**Comment by kensipe** [comment]
@alenkacz I continue to use double spaces after periods which I known from the past the karsten would prefer not to... but I prefer it and it doesn't make a difference in source or rendering into html.

File: docs/docs/checks.md (hunk 1 above)
Comment (kensipe) [reply docs/docs/checks.md]: @timcharper this consistent with [readiness check docs](https://mesosphere.github.io/marathon/docs/readiness-checks.html) and helps to set context for user against features they are already familiar with... if they are not... links are provided.

File: docs/docs/checks.md [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+    "intervalSeconds": 60,
+    "timeoutSeconds": 20
+ }
+}
+```
+
+### Example Command Check Usage
+
+```json
+"check":
```
[the remark this answers is not in the document]
Comment (kensipe) [reply docs/docs/checks.md]: no... check needs to be there... this is consistent with health checks and readiness checks.  it is the defining element of the app def

File: docs/docs/checks.md [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+  "cpus": 1,
+  "mem": 128,
+  "disk": 0,
+  "instances": 1,
+  "check": {
+    "exec": {
+      "command": {
+        "shell": "ls"
+      },
+   "intervalSeconds": 60,
```
[the remark this answers is not in the document]
Comment (kensipe) [reply docs/docs/checks.md]: man... thank you!   this json was wrong... it is fixed now

File: docs/docs/checks.md (hunk 1 above)
Comment (timcharper) [reply docs/docs/checks.md]: Okay. It's just that it will read weird a year in the future perhaps?

**Pull request merged**